content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -54,12 +54,19 @@ Choose this option for:
 * Headless services, or
 * `StatefulSets`
 
-To install Istio without mutual TLS authentication between sidecars:
+To install Istio with mutual TLS enabled and set to use permissive mode
+between sidecars:
 
 {{< text bash >}}
 $ kubectl apply -f install/kubernetes/istio-demo.yaml
 {{< /text >}}
 
+In this option, all services, as servers, can accept both plain text and
```

Comment:
all services, as servers  --> all server-side workloads?
Same to the following line.

Reply:
if so, we will need to have "client-side workloads will send plain text traffic" in the below sentence. I am afraid the 'service-side workloads' and 'client-side workloads' concepts might confuse audience